public/app/features/admin/ServerStats.tsx

```diff
@@ -1,79 +1,166 @@
-import React, { PureComponent } from 'react';
+import React, { useEffect, useState } from 'react';
 import { hot } from 'react-hot-loader';
 import { connect } from 'react-redux';
-import { Icon, Tooltip } from '@grafana/ui';
-import { NavModel } from '@grafana/data';
-import { StoreState } from 'app/types';
+import { css } from '@emotion/css';
+import { CardContainer, LinkButton, useStyles2 } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { AccessControlAction, StoreState } from 'app/types';
 import { getNavModel } from 'app/core/selectors/navModel';
-import Page from 'app/core/components/Page/Page';
 import { getServerStats, ServerStat } from './state/apis';
+import { contextSrv } from '../../core/services/context_srv';
 
-interface Props {
-  navModel: NavModel;
-  getServerStats: () => Promise<ServerStat[]>;
+export interface Props {
+  getServerStats: () => Promise<ServerStat | null>;
 }
 
-interface State {
-  stats: ServerStat[];
-  isLoading: boolean;
-}
+export const ServerStats = ({ getServerStats }: Props) => {
+  const [stats, setStats] = useState<ServerStat | null>(null);
+  const [isLoading, setIsLoading] = useState(true);
+  const styles = useStyles2(getStyles);
 
-export class ServerStats extends PureComponent<Props, State> {
-  state: State = {
-    stats: [],
-    isLoading: true,
-  };
+  useEffect(() => {
+    getServerStats().then(setStats);
+    setIsLoading(false);
+  }, [getServerStats]);
```

Comment:
Can you add a gif showing how the UI works in slow network scenario? Feels like the license info will be shifted when the stats finish loading. Would be great if we could have some kind of a [skeleton loader](https://www.cssscript.com/skeleton-loader-placeholder/) on the actual stat values, so that we avoid shifting the UI blocks when the content loads.

Reply:
Hmm, we could show the loader in the container with the same height as the cards with the data, so there's no UI shift after the data is loaded. Not sure if it's worth it to load additional assets such as skeletons just for this page, although it does look nice.

Reply:
This is what I meant: 

https://user-images.githubusercontent.com/8878045/130577577-706d5fc2-9013-415f-b4fc-ad8ab2fa1b16.mov

Reply:
Looks good to me, would only change the copy of the loading indicator to be "Loading instance stats..." or sth along these lines.

Reply:
Done.